src/modules/Touchdown/components/DailyHistory.js

```diff
@@ -0,0 +1,263 @@
+          className: 'hide-col-divider reserve-amt',
+        },
+        {
+          title: 'Winners',
+          dataIndex: 'sixForSevenWinnersCount',
+          key: 'sixForSevenWinnersCount',
+          className: 'winners col-divider colspan',
+          render: renderTextOnCurrentWeekCell,
+        },
+      ],
+    },
+    {
+      title: 'Weekly Reserve',
+      key: 'actualWeeklyReserve',
+      dataIndex: 'actualWeeklyReserve',
+      className: 'weekly-reserve',
+    },
+    {
+      title: 'TopProp Vig',
+      dataIndex: 'topPropVig',
+      key: 'topPropVig',
+      sorter: (a, b) => a.topPropVig - b.topPropVig,
+    },
+    {
+      title: 'Money Added By TopProp',
+      dataIndex: 'addedByTopProp',
+      key: 'addedByTopProp',
+      render: renderTextOnCurrentWeekCell,
+      sorter: (a, b) => a.addedByTopProp - b.addedByTopProp,
+    },
+    {
+      title: 'Profit/Loss',
+      dataIndex: 'profit',
+      key: 'profit',
+      render: renderTextOnCurrentWeekCell,
+      sorter: (a, b) => a.profit - b.profit,
+    },
+    {
+      title: 'Created Contest',
+      key: 'createdContest',
+      className: 'col-divider',
+      children: [
+        {
+          title: 'NBA',
+          dataIndex: 'nbaContestsCount',
+          key: 'nbaContestsCount',
+          className: 'hide-col-divider',
+        },
+        {
+          title: 'MLB',
+          dataIndex: 'mlbContestsCount',
+          key: 'mlbContestsCount',
+          className: 'hide-col-divider',
+        },
+        {
+          title: 'Soccer',
+          dataIndex: 'soccerContestsCount',
+          key: 'soccerContestsCount',
+          className: 'col-divider colspan',
+        },
+      ],
+    },
+    {
+      className: 'leaderboard-btn',
+      title: 'Daily Leaderboard',
+      dataIndex: 'prizePoolId',
+      key: 'prizePoolId',
+      render: renderLeaderboardButton,
+      onCell: (record) => ({
+        onClick: () => {
+          setSelectedLeaderboard({ ...record });
+        },
+      }),
+    },
+  ];
+  const formatAmount = (amount) => `${parseFloat(amount).toFixed(2)}`;
+
+  const prizePoolAmtWithDollar = (pp) =>
+    pp.map((data) => ({
+      ...data,
+      date: dayjs(data.startDate).format(MM_DD_YYYY),
+      actualJackpot: '$'.concat(formatAmount(data.actualJackpotAmount || 0)),
+
+      actualSixForSeven: '$'.concat(
+        formatAmount(data?.actualReserveAmount?.SIX_FOR_SEVEN || 0),
+      ),
+
+      profit: '$'.concat(data.profit),
+
+      actualWeeklyReserve: '$'.concat(
+        formatAmount(data?.actualWeeklyReserveAmount || 0),
+      ),
+
+      addedByTopProp: '$'.concat(
+        formatAmount(data?.actualWeeklyReserveAmount || 0),
+      ),
+
+      actualPrizePool: '$'.concat(formatAmount(data?.actualPrizePool)),
+    }));
+
+  const fetchDailyHistoryDate = () => {
+    setLoading(true);
+    getDailyTouchdownHistory({ variables: { startDate, endDate } })
+      .then(({ data }) => {

```

Comment:
if startDate, endDate used only here then remove from above and define startdate and enddate here only ,